libraries/lib-datahandler/src/main/java/org/eclipse/sw360/datahandler/couchdb/lucene/LuceneAwareDatabaseConnector.java

```diff
@@ -162,15 +162,16 @@ public void setResultLimit(int limit) {
 
             final Set<String> filterSet =restriction.getValue();
 
-            if(!filterSet.isEmpty()) {
-                final String fieldname = restriction.getKey();
-                String subQuery = formatSubquery(filterSet, fieldname);
+            if (!filterSet.isEmpty()) {
+                final String fieldName = restriction.getKey();
+                String subQuery = formatSubquery(filterSet, fieldName);
                 subQueries.add(subQuery);
             }
         }
 
-        if(!isNullOrEmpty(text))
-            subQueries.add( text + "*");
+        if (!isNullOrEmpty(text)) {
+            subQueries.add(text + "*");
```

Comment:
Shouldn't this line call `prepareWildcardQuery()`?

Reply:
added `prepareWildcardQuery` to this line